examples/tacopie/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2016, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-tacopie)
+
+# download tacopie
+hunter_add_package(tacopie)
+
+# now tacopie can be used
+find_package(tacopie CONFIG REQUIRED)
```

Comment:
> Don't I need to have a forked tacopie repo to install the tacopieConfig.cmake file?
> How can I achieve this using only the hunter.cmake file in the hunter repo?

It should be implemented in CMakeLists.txt of package itself, not in `hunter.cmake`.
I've created fork https://github.com/hunter-packages/tacopie for now but in general it belongs to  upstream (it's not "hunterization").

> I'm still quite new to this

If you read the docs from above you can find this template code:
* https://github.com/forexample/package-example/blob/cf2ea1d6a209fb9eca2ab83fdd0ac15fe4d3e807/Foo/CMakeLists.txt#L63-L147

Adapt the code and do not forget `cmake/Config.cmake.in`, should be enough.